frontend/src/pages/edge/EmptyEdgeModels.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import {
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  PageSection,
+  Title,
+} from '@patternfly/react-core';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+
+const EmptyEdgeModels: React.FC = () => (
+  <PageSection isFilled>
+    <EmptyState>
+      <EmptyStateIcon icon={PlusCircleIcon} />
+      <Title headingLevel="h1" size="lg" style={{ marginBottom: '10px' }}>
+        No models added
+      </Title>
+      <EmptyStateBody style={{ margin: 'auto', width: '50%' }}>
```

Comment:
Avoid all custom styles. We must adhere to what PF provides or updates to PF will create inconsistencies in our application.

```suggestion
    <EmptyState variant={EmptyStateVariant.large}>
      <EmptyStateIcon icon={PlusCircleIcon} />
      <Title headingLevel="h1" size="lg">
        No models added
      </Title>
      <EmptyStateBody>
```